pkg/daemons/control/server.go

```diff
@@ -427,7 +427,7 @@ func checkForCloudControllerPrivileges(runtime *config.ControlRuntime) error {
 		return err
 	}
 	crb := rbac.NewFactoryFromConfigOrDie(restConfig).Rbac().V1().ClusterRoleBinding()
-	_, err = crb.Get("cloud-controller-manager", metav1.GetOptions{})
+	_, err = crb.Get("k3s-cloud-controller-manager", metav1.GetOptions{})
```

Comment:
We try to use version.Program or version.ProgramUpper instead of using the `k3s` string literal, as this white-labeling allows rke2 to not have references to k3s all over the place. We aren't quite 100% on this yet, but that's the convention we're going for.

```suggestion
	_, err = crb.Get(version.Program + "-cloud-controller-manager", metav1.GetOptions{})
```